### PR TITLE
feat: apis to update profile

### PR DESCRIPTION
**Pull request by nwjbrandon**
- added apis to update care takers, pet owners, and administrators
- added apis to approved administrator accounts

### CODE REVIEW

File: tests/User/UserLoginController.test.js [hunk 1]
```diff
@@ -63,29 +64,10 @@ describe('Test UserLogin Controller', () => {
...
-      password,
-      role,
-    });
-    const decodedToken = DecodeAccessToken(res.body.accessToken);
-    Assert.deepStrictEqual(
-      {
-        email,
-        role,
-      },
-      _.omit(decodedToken, ['uid', 'iat']),
```
Comment (ambhinav) [inline tests/User/UserLoginController.test.js:81]: So `acessTokens` have `uid` and `iat` atrributes? Just to confirm that these aren't the `uid` of `CareTaker`, `PetOwner` etc... right?

File: src/User/UserService.js [hunk 2]
```diff
@@ -39,31 +43,102 @@ const UserLogin = async ({email, password, role}) => {
     users = await pool.query(SQLQueries.SELECT_CARE_TAKER, [email]);
   } else if (role === RoleUtils.PET_OWNER) {
     users = await pool.query(SQLQueries.SELECT_PET_OWNER, [email]);
+  } else if (role === RoleUtils.ADMINISTRATOR) {
+    users = await pool.query(SQLQueries.SELECT_ADMINISTRATOR, [email]);
```
Comment (ambhinav) [inline src/User/UserService.js:47]: Need an extra check to make sure the admin `is_approved` is `true` as well.

File: tests/User/UserApproveController.test.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    await pool.query('DELETE FROM psc_administrators');
+  });
+
+  afterEach('UserApproveController afterEach', async () => {
+    await pool.query('DELETE FROM care_takers');
+    await pool.query('DELETE FROM pet_owners');
+    await pool.query('DELETE FROM psc_administrators');
+  });
+
+  it('Should delete administrator', async () => {
```
Comment (ambhinav) [inline tests/User/UserApproveController.test.js]: I think the test name is wrong, this test is for fetching an **approved administrator**

File: tests/User/UserLoginController.test.js (hunk 1 above)
Comment (nwjbrandon) [reply tests/User/UserLoginController.test.js:81]: Yup `uid` is a mistake, and has been removed.

`iat` is by default added when you create the jwt, so need to remove before asserting in test cases.

File: src/User/UserService.js (hunk 2 above)
Comment (nwjbrandon) [reply src/User/UserService.js:47]: Done

File: tests/User/UserApproveController.test.js (hunk 3 above)
Comment (nwjbrandon) [reply tests/User/UserApproveController.test.js]: Done

**Review by ambhinav — APPROVED** [review]

**Pull request merged**